ci/jenkins/test.sh

```diff
@@ -810,6 +829,20 @@ function run_conformance {
     kubectl rollout status daemonset/antrea-agent -n kube-system
 
     set +e
+    # Conformance DNS test cases fail in ipv6 Kind cluster because here CoreDNS pods are IPV6 pods and 
+    # it can't reach IPV4 upstream namespace server.
+    # Note:Default CoreDNS ConfigMap uses /etc/resolv.conf to get upstream nameservers and the upstream nameservers can be accessed by IPV4
+    # but single stack IPV6 CoreDNS would not be able to connect that server and it return servfail, 
+    # to reolve this we need to forward DNS query to "vmware.com:53" and 
+    # it will return 'NXDOMAIN' so that client keep searching.
+    # For more detail: https://github.com/kubernetes/kubernetes/issues/94794 
+    if [[ $ENABLE_IPV6 == "ipv6" && "$TESTCASE" =~ "conformance" && $TESTBED_TYPE == "kind" && $JENKINS_LOCATION == "lab" ]]; then
```

Comment:
I am not sure how `JENKINS_LOCATION` can be processed during running jobs, as it should only be available when uploading jenkins jobs. Please correct me if I am wrong.